css/mirador.less/panels/bottom-panel.less

```diff
@@ -38,6 +38,12 @@
         padding: 0px 10px 0px 10px;
         /*transition: all 0.03s ease-out;*/
         display: inline-block;
+        margin-bottom: 11px !important;
+
+        &.highlight {
+          padding-left: 8px !important;
+          padding-right: 8px !important;
```

Comment:
Cool. Thanks for the update. The diff didn't make clear that those were pre-existing.